src/curve_fitting.vala

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+using Adw;
```

Comment:
were not using Adw here, or am I wrong?

Reply:
Correct, not sure why that was there tbh. Probably left-over from using another class as template.